app/controllers/admins/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Admins::UsersController < ApplicationController
```

Comment:
Have these inherit from `AdminsController` instead of `ApplicationController`. Then you don't need to repeat the private method `authorized?` or the `before_filter :authorized?`